### PR TITLE
fix_category

### PR DESCRIPTION
**Pull request by khieutrung12**
## Related Tickets:

-   [#47043](https://edu-redmine.sun-asterisk.vn/projects/ecommerce-intern-hn/issues/47043)

## WHAT this PR does?

- Edit system relationship between category and product
- Multiple select categories

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ready

**Comment by khieutrung12** [comment]
ready

File: app/Models/ProductCategory.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public $table = 'product_category';
+
+    protected $fillable = [
+        'product_id',
+        'category_id',
+    ];
+
+    protected $dates = [
+        'created_at',
+        'updated_at',
```
Comment (huyenttm-0901) [inline app/Models/ProductCategory.php]: sao phải tạo model cho bản pivot vậy em?

**Comment by khieutrung12** [comment]
ready

**Review by DTHNhung — APPROVED** [review]

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**